src/js/models/maps/assets/CesiumVectorData.js

```diff
@@ -16,9 +16,33 @@ define([
   VectorFilters
 ) {
   // Source: https://fontawesome.com/v6/icons/location-dot?f=classic&s=solid
-  // !Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.
-  const PIN_SVG_STRING = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 384 512'><path d='M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z'/></svg>";
+  const PIN_SVG_STRING = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512"><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z"/></svg>';
+  const PIN_OUTLINE_WIDTH = 30; // The width of the stroke around the pin is relative to the viewBox
+  const PIN_OUTLINE_COLOR = "white";
   const B64_START = 'data:image/svg+xml;base64,';
+  const PIN_SVG = convertFontAwesomeSvgForCesium(PIN_SVG_STRING, PIN_OUTLINE_WIDTH, PIN_OUTLINE_COLOR);
+
+  // Convert a Font Awesome SVG string to a SVG element that can be used as an image for a Cesium billboard
+  function convertFontAwesomeSvgForCesium(svgString, strokeWidth, strokeColor) {
+
+    const domParser = new DOMParser();
+    const svgElement = domParser.parseFromString(svgString, "image/svg+xml").querySelector("svg");
+
+    svgElement.removeChild(svgElement.firstChild);
```

Comment:
this makes me nervous🫣 I feel like I'd rather include an extra comment if people don't care to delete it, than accidentally removing a data path